### PR TITLE
Make the history and completion lists properly handle overflows

### PR DESCRIPTION
**Pull request by victorporof**
Fixes https://github.com/mozilla/tofino/issues/1194

Signed-off-by: Victor Porof <vporof@mozilla.com>

### CODE REVIEW

File: app/ui/shared/widgets/dropdown-menu-btn.jsx [hunk 1]
```diff
@@ -114,10 +114,10 @@ class DropdownMenuBtn extends Component {
         onMouseDown={this.handeButtonMouseDown}
         onMouseUp={this.handeButtonMouseUp}
         onClick={this.handleButtonClick}>
-        <SelectionList className={SELECTION_LIST_STYLE}
+        <SelectionList className={`${SELECTION_LIST_STYLE} ${this.props.dropdownListClassName || ''}`}
```
Comment (jsantell) [inline app/ui/shared/widgets/dropdown-menu-btn.jsx]: Should set default props for `this.props.dropdownListClassName`

File: app/ui/shared/widgets/autocompleted-search.jsx [hunk 2]
```diff
@@ -133,7 +134,7 @@ class AutocompletedSearch extends Component {
         ref={e => this.inputbar = e}
         onChange={this.handleInputChange}
         onKeyDown={this.handleInputKeyDown}>
-        <SelectionList className={SELECTION_LIST_STYLE}
+        <SelectionList className={`${SELECTION_LIST_STYLE} ${this.props.dropdownListClassName || ''}`}
```
Comment (jsantell) [inline app/ui/shared/widgets/autocompleted-search.jsx]: Should set default props for `this.props.dropdownListClassName`

File: app/ui/shared/widgets/selection-list.jsx [hunk 3]
```diff
@@ -39,7 +43,7 @@ class SelectionList extends Component {
   render() {
     return (
       <List {...omit(this.props, Object.keys(OmittedContainerProps))}
-        className={`${this.props.className || ''}`}>
+        className={`${SELECTION_LIST_STYLE} ${this.props.className || ''}`}>
```
Comment (jsantell) [inline app/ui/shared/widgets/selection-list.jsx]: Default className

**Comment by jsantell** [comment]
Just noticed this is still in progress, r+ if you don't have anything else!

**Comment by victorporof** [comment]
Not sure why this was labeled as "in progress". It's finished.

**Pull request merged**